### PR TITLE
Add support for dynamic sidebars for regions.

### PR DESCRIPTION
**Pull request by keyurva**
[sidebar-regions.webm](https://github.com/datacommonsorg/website/assets/1221814/ed73608f-e66d-4cb9-a222-aab44bab6b0c)

### CODE REVIEW

File: experimental/sdg-tracker/src/state/index.ts [hunk 1]
```diff
@@ -52,6 +52,42 @@ const MenuImageIcon = styled.img`
...
+
+const CONTINENTS = new Set([
+  "africa",
+  "antartica",
+  "asia",
+  "europe",
+  "northamerica",
+  "oceania",
+]);
+
```
Comment (dwnoble) [inline experimental/sdg-tracker/src/state/index.ts:90]: Can you move these constants to https://github.com/datacommonsorg/website/blob/62d97f294f5d1ce229443bf70eb40826caf1fb82/experimental/sdg-tracker/src/utils/constants.ts

(also can you use the existing "Earth" constant here https://github.com/datacommonsorg/website/blob/62d97f294f5d1ce229443bf70eb40826caf1fb82/experimental/sdg-tracker/src/utils/constants.ts#L22)

File: experimental/sdg-tracker/src/utils/DataCommonsClient.ts [hunk 2]
```diff
@@ -106,5 +106,17 @@ class DataCommonsClient {
     });
     return (await response.json()) as BulkObservationExistenceResponse;
   }
+
+  async getCountriesInRegion(regionDcid: string): Promise<string[]> {
+    const url = `${this.apiRoot}/api/place/descendent?descendentType=Country&dcids=${regionDcid}`;
```
Comment (dwnoble) [inline experimental/sdg-tracker/src/utils/DataCommonsClient.ts:111]: use Country constant https://github.com/datacommonsorg/website/blob/62d97f294f5d1ce229443bf70eb40826caf1fb82/experimental/sdg-tracker/src/utils/constants.ts#L32

**Review by dwnoble — APPROVED** [review]
Approved pending above changes!

File: experimental/sdg-tracker/src/state/index.ts (hunk 1 above)
Comment (keyurva) [reply experimental/sdg-tracker/src/state/index.ts]: Done.

File: experimental/sdg-tracker/src/utils/DataCommonsClient.ts (hunk 2 above)
Comment (keyurva) [reply experimental/sdg-tracker/src/utils/DataCommonsClient.ts]: Done.

**Comment by keyurva** [comment]
Thanks Dan.

**Pull request merged**